src/ptvsd/_vendored/pydevd/_pydevd_bundle/pydevd_comm.py

```diff
@@ -982,7 +982,7 @@ def __create_frame():
         is_error = isinstance(eval_result, ExceptionOnEvaluate)
         if is_error:
             if context == 'hover':  # In a hover it doesn't make sense to do an exec.
-                _evaluate_response(py_db, request, result='')
+                _evaluate_response(py_db, request, result='', error_message='Exception evaluating.')
```

Comment:
The error message is unclear - should be something like "Exception occurred during evaluation".